### PR TITLE
Stop Page Jump on UK Network Front (Part 2)

### PR DESCRIPTION
**Pull request by kelvin-chappell**
This fixes the size of the top-above-nav slot for the three expected ad heights.

It continues from #9707.

This first iteration uses manual switches to set the size of the slot but it will be set automatically in the next iteration.

/cc @kenlim @tijanamiletic @Calanthe

### CODE REVIEW

File: common/test/views/support/CommercialTest.scala [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    FixedTopAboveNavAdSlotSwitch.switchOn()
+    topAboveNavSlot.cssClasses(metaDataFromId("us")) should
+      endWith("top-banner-ad-container--reveal")
+    topAboveNavSlot.cssClasses(metaDataFromId("uk/culture")) should
+      endWith("top-banner-ad-container--reveal")
+    topAboveNavSlot.cssClasses(metaDataFromId(
+      "business/2015/jul/07/eurozone-calls-on-athens-to-get-serious-over-greece-debt-crisis"))
+      .should(endWith("top-banner-ad-container--reveal"))
+  }
+}
```
Comment (Calanthe) [inline common/test/views/support/CommercialTest.scala]: I like the test coverage!

**Comment by Calanthe** [comment]
Here https://github.com/guardian/frontend/pull/9707 Rich suggested that we should remove:
'.top-banner-ad-container--reveal iframe {
  transform: translateZ(0);
  transition: height 1s cubic-bezier(0, 0, 0, .985);
}'

Can you do that in this PR?

**Comment by kelvin-chappell** [comment]
@Calanthe We still need it for every other page apart from /uk.  This PR is only for one URL.

**Comment by Calanthe** [comment]
That's true. Good point!

We still should remember to remove it later.

**Comment by kelvin-chappell** [comment]
I've added another switch for the 1x1 case as it would have opened the slot to its large state rather than closing the ad slot altogether.  Although I now realise that in that case we should just omit the ad slot from the page rather than give it a special class.

**Comment by Calanthe** [comment]
:+1:

**Pull request merged**